apps/platform/src/middleware.ts

```diff
@@ -0,0 +1,9 @@
+import createMiddleware from 'next-intl/middleware';
+import { routing } from './i18n/routing';
+
+export default createMiddleware(routing);
+console.log("hello");
```

Comment:
I understand this is required for debugging, but please add a todo to remove this log and remove it once it is no longer needed. This could explode the prod logs